### DISCUSSION TITLE
面积图的area:click 事件返回的数据是整一区域的数据，是否有方法能获取点击点的单条数据

### DISCUSSION THREAD

**Discussion by zhouhe-1476452306** [antvis/G2#5199] [Q&A]

**Comment by pearmini** [comment]
暂时没有提供这个方法，可以拿到点击数据，然后用 coordinate 和 scale invert 数据。

**Reply by xyr550** [reply]
大佬，具体怎么转化，能不能详细说下

**Reply by zhouhe-1476452306** [reply]
根据点击点的位置信息通过chart.getCoordinate().invert()获取到归一化之后的结果
在通过scaleX去invert 点击点周围的数据 只能获取到横坐标，但是当存在2组及以上数据时，怎样去获取点击的属于哪一组，这个应该在area:click的点击事件中没有相关的参数，请问这种情况要怎么处理呀

**Reply by zhouhe-1476452306** [reply]
这种情况怎样区分点击的是那个区域的数据？
![image](https://github.com/antvis/G2/assets/75061709/3dbb6895-472a-4532-a34e-bfe5ec974cf7)

**Reply by pearmini** [reply]
是重叠面积图的情况吗？

**Reply by zhouhe-1476452306** [reply]
这个是当出现多组数据类似这种数据
![image](https://github.com/antvis/G2/assets/75061709/222b17c6-61d8-480a-b11c-f65a4b8f0bd7)
根据上面的计算方式可以算出点击横坐标索引值， 面积图area:click返回的是全部数据，无法区分点击的是value or value1目前的解决方案是通过每一个chart.area的数据添加key值去区分是哪一组的value
重叠的话确实点击获取的是层级高的那一组数据
